pkg/compiler/builtin_test.go

```diff
@@ -0,0 +1,111 @@
+package compiler_test
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/compiler"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMaxMin(t *testing.T) {
+	tests := []struct {
+		name        string
+		args        []stackitem.Item
+		expected    map[string]*big.Int
+		expectedErr map[string]string
+		srcTemplate string
+	}{
+		{
+			name: "invalid type named",
+			expectedErr: map[string]string{
+				"max": "only integer types are supported by max",
+				"min": "only integer types are supported by min",
+			},
```

Comment:
Use a single string with `%s` instead of min/max and instead of the map. Format this string in-lace in test, like it's done for the program template.